ci/matrix.xml

```diff
@@ -225,6 +228,8 @@
         javalibExTestSuite/test:scalastyle helloworld/scalastyle \
         reversi/scalastyle testingExample/scalastyle \
         testingExample/test:scalastyle
+    sbt ++$scala ir/compile:doc tools/compile:doc jsEnvs/compile:doc \
```

Comment:
I'm ready to add those if needed, but before patch, the tools-cli-stubs-sbtplugin task already contained several not chained sbt commands (not separated by &&); I just copied the existing behavior. 
If needed I can also add "&&" between the mima and scalastyle tests.